### PR TITLE
mark informative references as informative and fix broken references

### LINKED ISSUE #16: Broken references in SPARQL 1.2 Entailment Regimes
While crawling [SPARQL 1.2 Entailment Regimes](https://w3c.github.io/sparql-entailment/spec/), the following links to other specifications were detected as pointing to non-existing anchors:
* [x] https://w3c.github.io/rdf-semantics/spec/#RDFRules
* [x] https://w3c.github.io/rdf-semantics/spec/#RDFSRules
* [x] https://w3c.github.io/rdf-semantics/spec/#glossSkolemization
* [x] https://w3c.github.io/rdf-semantics/spec/#RDFINTERP
* [x] https://w3c.github.io/rdf-semantics/spec/#defallocated
* [x] https://w3c.github.io/rdf-semantics/spec/#RDFSINTERP
* [x] https://w3c.github.io/rdf-semantics/spec/#DTYPEINTERP

<sub>This issue was detected and reported semi-automatically by [Strudy](https://github.com/w3c/strudy/) based on data collected in [webref](https://github.com/w3c/webref/).</sub>

### PR DESCRIPTION
**Pull request by pfps**
Fixes #16

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/pull/17.html" title="Last updated on Apr 12, 2023, 5:55 PM UTC (0f7115b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/17/b2bea68...0f7115b.html" title="Last updated on Apr 12, 2023, 5:55 PM UTC (0f7115b)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -660,10 +660,10 @@ <h2>RDF Entailment Regime</h2>
...
-                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#glossSkolemization">Skolemization</a> of SG with
+                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#skolemization">Skolemization</a> of SG with
                 respect to a vocabulary disjoint from the vocabulary of SG and BGP. Applying sk to a term t, written sk(t), yields sk(t) if sk is defined for t and t otherwise; applying sk to a BGP,
                 written sk(BGP), replaces each blank node b in BGP for which sk is defined with sk(b). The set rdfV contains the URI references of the
-                <a data-cite="RDF12-SEMANTICS#RDFINTERP">RDF vocabulary</a> and rdfV-Minus is the set of URI references in rdfV minus URI references of the form <code>rdf:_n</code>
+                <a data-cite="RDF12-SEMANTICS#RDF vocabulary">RDF vocabulary</a> and rdfV-Minus is the set of URI references in rdfV minus URI references of the form <code>rdf:_n</code>
```
Comment (TallTed) [inline spec/index.html:666]: `RDF12-SEMANTICS#RDF vocabulary` is an invalid URI... I looked at [RDF12-SEMANTICS](https://w3c.github.io/rdf-semantics/spec/), but I was not able to figure out what this fragment ID should be.

File: spec/index.html [hunk 2]
```diff
@@ -1046,19 +1045,19 @@ <h2>RDFS Entailment Regime</h2>
...
-                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#glossSkolemization">Skolemization</a> of SG with
+                  <a data-cite="SPARQL12-QUERY#BGPsparqlBNodes">scoping graph</a> for G and BGP, sk(SG) a <a data-cite="RDF12-SEMANTICS#skolemization">Skolemization</a> of SG with
                 respect to a vocabulary disjoint from the vocabulary of SG and BGP. Applying sk to a term t, written sk(t), yields sk(t) if sk is defined for t and t otherwise; applying sk to a BGP,
                 written sk(BGP), replaces each blank node b in BGP for which sk is defined with sk(b). The set rdfsV contains the URI references of the
-                <a data-cite="RDF12-SEMANTICS#RDFSINTERP">RDFS vocabulary</a> and rdfsV-Minus is the set of URI references in rdfsV minus URI references of the form
+                <a data-cite="RDF12-SEMANTICS#RDFS vocabulary">RDFS vocabulary</a> and rdfsV-Minus is the set of URI references in rdfsV minus URI references of the form
```
Comment (TallTed) [inline spec/index.html:1060]: As above, `RDF12-SEMANTICS#RDFS vocabulary` is not a valid URI.

Comment (pfps) [reply spec/index.html:1060]: #dfn-rdfs-vocabulary, fixed (soon)

File: spec/index.html (hunk 1 above)
Comment (pfps) [reply spec/index.html:666]: @dfn-rdf-vocabulary, but in a PR, fixed (soon)

**Comment by pfps** [comment]
Fixed two incorrect links.

**Pull request merged**